src/js/stream.js

```diff
@@ -31,21 +42,25 @@ class Stream {
 		}
 	}
 
-	getJsonWebToken () {
-		return fetch('https://comments-api.ft.com/user/auth/', {
+	getJsonWebToken (displayName) {
+		const url = displayName ?
```

Comment:
I think during the merging of branch `v6` in to this branch, this logic is lost:

https://github.com/Financial-Times/o-comments/blob/11318be3fa48c4dcd1841d10d30bcde1b5cc9e1b/src/js/stream.js#L35

Reply:
I've re-added it!

Reply:
👍